src/SDKs/Azure.Core/data-plane/Azure.Core.Tests/ClientTestBase.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Reflection;
+using Castle.DynamicProxy;
+using NUnit.Framework;
+
+namespace Azure.Core.Tests
+{
+    [TestFixture(true)]
+    [TestFixture(false)]
+    public class ClientTestBase
```

Comment:
It'd probably be good to add a comment that this forces both sync and async tests to run in separate passes because at first I thought it was a typo.  If we set them here on the base class, can I override whether they're included on a derived class in case I don't want one of them to run for some reason?

Reply:
>  If we set them here on the base class, can I override whether they're included on a derived class in case I don't want one of them to run for some reason?

Not really, but why would you use this base type but prefer to avoid running both kinds of tests.

Reply:
Because as soon as you add a base class, people are going to start filling it with goodies.  I may want some of your goodies without running async tests.  It's fine to wait until that becomes an issue to decide what to do here.

Reply:
Let's wait to see where it goes, I have plans to also layer recorded tests onto this.